hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -70,13 +70,28 @@ E.g. 0x21 means try SD first followed by network boot then stop. Where as 0x2 wo
 The retry counters are reset when switching to the next boot mode.
 
 BOOT_ORDER fields  
+The BOOT_ORDER field defines sequence for the different boot modes. It is read right to left and up to 8 digits may be defined.
+
 * 0x0 - NONE (stop with error pattern)  
 * 0x1 - SD CARD  
 * 0x2 - NETWORK  
+* 0x3 - USB device boot (usbboot)[https://github.com/raspberrypi/usbboot] - compute module only.
```

Comment:
Capital letters on Compute Module ?

Reply:
Didn't know that there is a Compute Module with bcm2711 ;)

When will it be released?